react-frontend/src/components/LogWorkout.js

```diff
@@ -105,9 +134,17 @@ function LogWorkout({ handleLogWorkoutSubmit, ...props }) {
                     <Offcanvas.Body style={{ paddingTop: "8px" }}>
                          <Form
                               noValidate
-                              // validated={validated}
+                              validated={validated}
                               onSubmit={handleSubmit}
                          >
+                              {/* Alert for displaying validation errors */}
```

Comment:
I like the alert but this is the only place where an alert is implemented. If we want to be consistent / less work as to not change all the other pages maybe we can just keep the input validation ui and not use an alert.

Reply:
so should i delete it?

Reply:
Follow up: asked in person and mentioned that it was ok to keep